virtual-host-gatherer/scripts/virtual-host-gatherer

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 #
 # Copyright (c) 2015 SUSE LLC. All Rights Reserved.
```

Comment:
While we're at it we could also fix this copyright line to include the last 6 years